server.go

```diff
@@ -20,6 +20,9 @@ import (
 const (
 	// SftpServerWorkerCount defines the number of workers for the SFTP server
 	SftpServerWorkerCount = 8
+
+	defaultFileMode = 0o644
+	defaultDirMode  = 0o750
```

Comment:
Nope, just says that defaults should be used: https://datatracker.ietf.org/doc/html/draft-ietf-secsh-filexfer-02#section-5

Reply:
Also note that the defaults are largely used for tests; the only case where the default file mode is used is for `client.Create`, because it calls `client.OpenFile` underneath (just like `os.Create`), it passes the default file mode. In all other cases the user passes the desired file mode.